verifiers/internal/gha/provenance_test.go

```diff
@@ -969,3 +970,35 @@ func Test_VerifyVersionedTag(t *testing.T) {
 		})
 	}
 }
+
+func Test_verifyBundle(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name     string
+		path     string
+		expected error
+	}{
+		{
+			name: "valid",
+			path: "./testdata/bundle/attestation.intoto.sigstore",
```

Comment:
we'll add invalid / adversarial ones inn the future?

Reply:
Just added one or two that were easily to manipulate without creating a test Rekor signer. Filed a follow-up issue 
https://github.com/slsa-framework/slsa-verifier/issues/461

We'll be generating a suite from sigstore-go pretty soon, so we can close this out either by borrowing cases there or moving the code to call a verification func there that's tested.